<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chiya</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>453</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="lcHorizontalLayout">
     <item>
      <widget class="QLabel" name="lcLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>This module switches layouts just like windows do, but with more customize options</string>
       </property>
       <property name="text">
        <string>Layout Controller</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="lcStateButton">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lcSettingsButton">
       <property name="toolTip">
        <string>Layout Controller settings</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>453</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="clHorizontalLayout">
     <item>
      <widget class="QLabel" name="clLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>This module switches layouts just like windows do, but with more customize options</string>
       </property>
       <property name="text">
        <string>Correct Layout</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clStateButton">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clSettingsButton">
       <property name="toolTip">
        <string>Layout Controller settings</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionMainSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionCheckForUpdates"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionMainSettings">
   <property name="text">
    <string>Main settings</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
